themes/finna2/templates/RecordTab/usercomments.phtml

```diff
@@ -4,55 +4,62 @@
   $user = $this->auth()->isLoggedIn();
   $ratings = $this->record($this->driver)->ratingAllowed();
   $allowCommenting = $user && $this->record($this->driver)->commentingAllowed($user);
+  $recordType = $this->driver->tryMethod('getRecordType');
 ?>
-<div class="comment-list">
-  <?=$this->render('record/comments-list.phtml', ['driver' => $this->driver])?>
-</div>
+<?php if ($recordType === 'lrmi' && $externalLink = $this->driver->getExternalLink()): ?>
```

Comment:
Käytä `$this->driver->tryMethod('getExternalLink')`. Ei tarvitse tarkistella muuta.